src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1445,23 +1387,6 @@ private IDataSource OpenDataSourceConnection(ConnectionInfo connInfo, string fea
             return null;
         }
 
-        /// <summary>
```

Comment:
We don't need ServerConnection because it's the Sql equivalent of the DataSource object that Kusto uses.